src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -26,13 +26,14 @@ public class WebHostRpcWorkerChannelManager : IWebHostRpcWorkerChannelManager
         private readonly ILoggerFactory _loggerFactory = null;
         private readonly IRpcWorkerChannelFactory _rpcWorkerChannelFactory;
         private readonly IMetricsLogger _metricsLogger;
+        private readonly IWorkerProfileManager _profileManager;
         private string _workerRuntime;
         private Action _shutdownStandbyWorkerChannels;
         private IConfiguration _config;
 
         private ConcurrentDictionary<string, Dictionary<string, TaskCompletionSource<IRpcWorkerChannel>>> _workerChannels = new ConcurrentDictionary<string, Dictionary<string, TaskCompletionSource<IRpcWorkerChannel>>>(StringComparer.OrdinalIgnoreCase);
 
-        public WebHostRpcWorkerChannelManager(IScriptEventManager eventManager, IEnvironment environment, ILoggerFactory loggerFactory, IRpcWorkerChannelFactory rpcWorkerChannelFactory, IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, IMetricsLogger metricsLogger, IOptionsMonitor<LanguageWorkerOptions> languageWorkerOptions, IConfiguration config)
+        public WebHostRpcWorkerChannelManager(IScriptEventManager eventManager, IEnvironment environment, ILoggerFactory loggerFactory, IRpcWorkerChannelFactory rpcWorkerChannelFactory, IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, IMetricsLogger metricsLogger, IOptionsMonitor<LanguageWorkerOptions> languageWorkerOptions, IConfiguration config, IWorkerProfileManager profileManager)
```

Comment:
nit: this is getting pretty long, lets break out each parameter into it's own line
```suggestion
        public WebHostRpcWorkerChannelManager(IScriptEventManager eventManager,
                                              IEnvironment environment,
                                              ILoggerFactory loggerFactory,
                                              IRpcWorkerChannelFactory rpcWorkerChannelFactory,
                                              IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions,
                                              IMetricsLogger metricsLogger,
                                              IOptionsMonitor<LanguageWorkerOptions> languageWorkerOptions,
                                              IConfiguration config,
                                              IWorkerProfileManager profileManager)
```